### PR TITLE
v8 - Major changes and versioning decisions

### LINKED ISSUE #225: Remove PIP install
The recommended usage pattern is to use CI-images as your build image.  Remove the pip install and provide instructions on how to install if not using CI-images.

### LINKED ISSUE #226: Remove NPM builds from rok8s-scripts
The recommended usage pattern is to use CI-images as your build image. Remove the npm install and provide instructions on how to install if not using CI-images.

### PR DESCRIPTION
**Pull request by sudermanjr**
Planned changelog:

```
v8.0.0

FUTURE BREAKING CHANGES - PLEASE READ

- Decided that future minor version releases may contain breaking changes introduced by a dependency.  Please review your pinning strategy when upgrading.

Changed

- Upgraded Helm to version 2.13.0 (breaking)
- Upgraded Kubectl to version 1.12.6 (potentially breaking if you are on an old kubernetes version)
- Upgraded Sops to version 3.2.0
- Removed the ability to `pip install` and removed npm publish - Please use ci-images from now on.
- Added a CONTRIBUTING doc
```

Also Fixes #225 and fixes #226

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Releasing New Versions of rok8s-scripts
 
 ## Before Merging the Pull Request!
-Please be sure to update `./setup.py` and `./package.json` to reflect the new version you wish to release. CODEOWNERS should be able to answer what the next version to release should be.
+Please be sure to update `./package.json` to reflect the new version you wish to release. CODEOWNERS should be able to answer what the next version to release should be.
```
Comment (robscott) [inline docs/releasing.md]: package.json has also been removed here

File: README.md [hunk 2]
```diff
@@ -25,6 +25,26 @@ Most notably, the CI example includes sample configuration for the following pla
 
 On their own, these examples may not make a lot of sense. There's a lot more documentation below that should cover everything included in these examples and more.
 
+## Versioning v8.0.0 and beyond
+
+Rok8s-scripts contains a number of dependencies that have various ways of versioning themselves.  Most notably, helm tends to break backward compatibility with every minor release.  We have decided that post v8 of rok8s-scripts, we will update our versions according to the version change of the underlying tool.  For example, if helm changes from `2.13.0` to `2.14.0`, we will change the version of rok8s scripts by one minor version.  This will be clearly mentioned in the release notes. This means that a minor version of rok8s-scripts could introduce breaking changes to the CI/CD pipelines that are using it.
```
Comment (robscott) [inline README.md]: I think Helm should be capitalized here. As a tiny personal preference, I also like single spaces to start a sentence, but that's far from necessary here.

File: docs/releasing.md (hunk 1 above)
Comment (sudermanjr) [reply docs/releasing.md]: I just removed this section.  I don't believe there is anywhere else that needs version updating.

File: README.md (hunk 2 above)
Comment (sudermanjr) [reply README.md]: Fixed

**Comment by sudermanjr** [comment]
> These changes look great! I think it would be good to update these [references to ci-images v7](https://github.com/reactiveops/rok8s-scripts/search?q=v7&unscoped_q=v7) to v8 as well, but that doesn't have to be part of this specific PR.

I went ahead and fixed them.

File: .gitignore [hunk 3]
```diff
@@ -2,3 +2,4 @@ package-lock.json
 *egg-info
 .vscode
 .DS_Store
+CHANGELOG
```
Comment (endzyme) [inline .gitignore]: I'm curious why we're ignoring CHANGELOG here. I didn't see and specific callout in the PR or the commits or Issues that I could find. There doesn't appear to be a CHANGELOG file today so I'm curious why we're specifically ignoring it.

File: .gitignore [hunk 4]
```diff
@@ -2,3 +2,4 @@ package-lock.json
 *egg-info
```
Comment (endzyme) [inline .gitignore]: Could egg and package lock be removed as a part of removing the npm and pip processes?

File: .gitignore (hunk 3 above)
Comment (sudermanjr) [reply .gitignore]: Ah, that was something I did early on and forgot to cleanup.  My bad.

File: .gitignore (hunk 4 above)
Comment (sudermanjr) [reply .gitignore]: yep.

**Pull request merged**